pkg/kubelet/apis/kubeletconfig/v1beta1/defaults.go

```diff
@@ -45,11 +43,32 @@ func addDefaultingFuncs(scheme *kruntime.Scheme) error {
 }
 
 func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
+	if obj.SyncFrequency == zeroDuration {
+		obj.SyncFrequency = metav1.Duration{Duration: 1 * time.Minute}
+	}
+	if obj.FileCheckFrequency == zeroDuration {
+		obj.FileCheckFrequency = metav1.Duration{Duration: 20 * time.Second}
+	}
+	if obj.HTTPCheckFrequency == zeroDuration {
+		obj.HTTPCheckFrequency = metav1.Duration{Duration: 20 * time.Second}
+	}
+	if obj.EnableServer == nil {
+		obj.EnableServer = utilpointer.BoolPtr(true)
+	}
```

Comment:
Do we need this field?

Reply:
Docs say you should set `--enable-server=false` if you also set `--runonce`. Though we had discussed whether runonce was still a useful feature...

Is there a case where you'd want to run standalone without the server enabled?